pallets/subtensor/tests/sudo.rs

```diff
@@ -51,37 +51,37 @@ fn test_sudo_registration() {
-		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 8).unwrap(), 8 );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(0), U256::from(0), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(1), U256::from(1), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(2), U256::from(2), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(3), U256::from(3), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(4), U256::from(4), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(5), U256::from(5), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(6), U256::from(6), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(7), U256::from(7), 10, 11) );
+        assert_ok!( SubtensorModule::sudo_register(<<Test as Config>::RuntimeOrigin>::root(), 0, U256::from(8), U256::from(8), 10, 11) );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(0) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(1) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(2) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(3) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(4) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(5) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(6) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(7) ), 11 );
+        assert_eq!( SubtensorModule::get_coldkey_balance( &U256::from(8) ), 11 );
+        assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 0).unwrap(), U256::from(0) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 1).unwrap(), U256::from(1) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 2).unwrap(), U256::from(2) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 3).unwrap(), U256::from(3) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 4).unwrap(), U256::from(4) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 5).unwrap(), U256::from(5) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 6).unwrap(), U256::from(6) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 7).unwrap(), U256::from(7) );
+		assert_eq!( SubtensorModule::get_hotkey_for_net_and_uid( 0, 8).unwrap(), U256::from(8) );
```

Comment:
out of curiosity... why is this tabbed?

Reply:
huum. in my vscode, I do not see any extra tabs and all lined are aligned. not sure why we see it different here.

Reply:
this is the interpretation though.... sounds like it depends on the editor,  there is  probably some difference there